tests/tests_integration/test_api/test_extraction_pipelines.py

```diff
@@ -33,6 +36,28 @@ def new_extpipe(cognite_client):
     assert cognite_client.extraction_pipelines.retrieve(extpipe.id) is None
 
 
+@pytest.fixture
```

Comment:
Let's add some broader scoping here to avoid re-running for each test using it

Reply:
It is only used once, and due to the scope of `new_extpipe`, this cannot be made broader without rewriting all the tests below.